pallets/maintenance-mode/src/lib.rs

```diff
@@ -0,0 +1,152 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+/// Edit this file to define custom logic or remove it if it is not needed.
+/// Learn more about FRAME and the core library of Substrate FRAME pallets:
+/// <https://docs.substrate.io/reference/frame-pallets/>
+pub use pallet::*;
+
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+pub mod weights;
+pub use weights::*;
+
+pub const ACTIVATE: bool = true;
+pub const DEACTIVATE: bool = false;
```

Comment:
I would use either a verb with past tense, or an adjective .. but not an infinitive verb, e.g. Activated or Active.